util/eventbus/bus.go

```diff
@@ -173,6 +180,27 @@ func (b *Bus) shouldPublish(t reflect.Type) bool {
 	return len(b.topics[t]) > 0
 }
 
+func (b *Bus) listClients() []*Client {
+	b.clientsMu.Lock()
+	defer b.clientsMu.Unlock()
+	return b.clients.Clone().Slice()
```

Comment:
As we are already locked here, do we need to clone? `Slice` allocates a new slice for us.